brave-core/src/main/java/com/github/kristofa/brave/LocalTracer.java

```diff
@@ -68,6 +68,13 @@ static Builder builder(LocalTracer source) {
     }
 
     /**
+     * Clears current span.
+     */
+    public void clearCurrentSpan() {
```

Comment:
i'm not sure why i'd want to use localSpanThreadBinder instead of localTracer here?

Given that the other tracers (serverTracer and clientTracer) have this method doesn't it make sense that localTracer also for the sake of consistency has it?

Then the effort to reduce API (/cognitive load) can be done separately, but equally to all three tracers?